src/wmtk/simplex/cofaces_single_dimension.hpp

```diff
@@ -3,12 +3,13 @@
 #include <wmtk/Primitive.hpp>
 #include <wmtk/Tuple.hpp>
 #include <wmtk/simplex/Simplex.hpp>
-
+#include <wmtk/simplex/SimplexCollection.hpp>
 namespace wmtk::simplex {
 
-    // Returns the cofaces of a provided simplex, but only providing the cofaces in the provided coface type
+// Returns the cofaces of a provided simplex, but only providing the cofaces in the provided coface
+// type
 
-std::vector<Tuple> cofaces_single_dimension_tuples(
+SimplexCollection cofaces_single_dimension(
```

Comment:
I'm going to leave everything that's related to cofaces_single_dimension unchanged for now, since Michael is going to possibly rewrite a lot about this function in https://github.com/wildmeshing/wildmeshing-toolkit/issues/514